docs/supported-components-data/12-restify-ubuntu-20.04-node-v12.22.6-details-2021-09-13T19_02_49.136Z.raw

```diff
@@ -752,9 +752,9 @@ metrics.stats
   active counts undefined
   i 130 b 152 a 152 d 113
 {}
-npm ERR! Linux 5.8.0-1039-azure
-npm ERR! argv "/__t/node/12.22.5/x64/bin/node" "/__w/appoptics-apm-node/appoptics-apm-node/node_modules/.bin/npm" "install" "--save-dev" "restify@6.2.1"
-npm ERR! node v12.22.5
+npm ERR! Linux 5.8.0-1040-azure
+npm ERR! argv "/__t/node/12.22.6/x64/bin/node" "/__w/appoptics-apm-node/appoptics-apm-node/node_modules/.bin/npm" "install" "--save-dev" "restify@6.2.1"
+npm ERR! node v12.22.6
 npm ERR! npm  v2.14.8
 
 npm ERR! primordials is not defined
```

Comment:
As a result of this error no json is being generated and as a result of that `restify` is not included in the final `.human` file. 

Investigating.